zipline/src/commonMain/kotlin/app/cash/zipline/DefaultZiplineSerializersModule.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2021 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.zipline
+
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.descriptors.SerialDescriptor
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+import kotlinx.serialization.modules.SerializersModule
+
+/**
+ * All Zipline [SerializersModule] should include this to get serialization coverage of core types
```

Comment:
Thinking about this... rather than requiring callers to remember to install our SerializerModule in their SerializerModule, I think the nicer API is to take the caller’s possibly-empty module, and install _that_ in a module that we define that has Throwable.

For follow-up.